src/combine.cpp

```diff
@@ -6469,6 +6469,12 @@ static void handle_oc_block_type(chunk_t *pc)
       chunk_t *apo = chunk_get_next_ncnl(tpc);   // arg open paren
       chunk_t *apc = chunk_skip_to_match(apo);   // arg close paren
 
+      // Check apo is '(' or else this might be a block literal.
```

Comment:
Please add a comment with the issue number

Reply:
Thanks for the review. Added.